deadline-monitor.py

```diff
@@ -42,7 +42,7 @@ def run(self):
             print "None found. Everything is OK."
             return
         for pod in deadlined_pods:
-            pv_check = self.report_pod_pv(pod)
```

Comment:
For now, pods in deadline state - we don't care.Even if we did report about deadline pods, ops will have to do same thing that this script does (i.e delete it).

But I am really hoping this is a temporary measure and we shouldn't be doing this long term.